docs/rfcs/rfc-018-mathematical-operators.md

```diff
@@ -0,0 +1,126 @@
+# Mathematical operators
+
+## Problem
+As described in the issue #677, if we have to do a mathematical operation using the GraphQL library, 
+the common approach is to query the database and then perform the mutation with the final property value.
+The former approach has several downsides:
+- It requires multiple roundtrips to the database.
+- It's not concurrency safe.
+- It requires more code than necessary in the client.
+
+## Proposed Solution
+Add new input properties that solve atomic operations like:
+- `_ADD`
+- `_SUBTRACT`
+- `_MULTIPLY`
+- `_DIVIDE`
+
+For instance `counter_ADD` or `counter_SUBTRACT`.
+These properties should be available for all the following types:
+- Int
+- Float
+- BigInt
+
+### Usage Examples
```

Comment:
Just for completion's sake (and a bit of nitpicking on my end): Shall we add the typeDefs here as well?

Reply:
I agree :)